ktor-http/common/src/io/ktor/http/HttpStatusCode.kt

```diff
@@ -103,21 +101,17 @@ public data class HttpStatusCode(val value: Int, val description: String) {
         public val VariantAlsoNegotiates: HttpStatusCode = HttpStatusCode(506, "Variant Also Negotiates")
         public val InsufficientStorage: HttpStatusCode = HttpStatusCode(507, "Insufficient Storage")
 
+        private val statusCodesMap: Map<Int, HttpStatusCode> = allStatusCodes().associateBy { it.value }
         /**
          * All known status codes
          */
-        public val allStatusCodes: List<HttpStatusCode> = allStatusCodes()
-
-        private val byValue: Array<HttpStatusCode?> = Array(1000) { idx ->
-            allStatusCodes.firstOrNull { it.value == idx }
-        }
+        public val allStatusCodes: List<HttpStatusCode> get() = statusCodesMap.values.toList()
```

Comment:
Please remove `get()`. Otherwise, every call to this property will create a new list.

Reply:
Done